### DISCUSSION TITLE
Is there an easy way to start test maps from the start?

### DISCUSSION THREAD

**Discussion by nobbele** [ppy/osu#27735] [Q&A]
In stable I'd just scroll forward a bit (to get to the last note) and click play to make it start from the beginning but in lazer it doesn't seem to do that. Is there another intended workflow or is that feature just missing?

**Comment by Walavouchey** [comment]
scroll to the beginning in the editor? play the map normally from song select?

**Reply by nobbele** [reply]
obviously, but I mean an *easy*/quick way

**Reply by Walavouchey** [reply]
they're both equally easy.

- try `Z`, `Z`, `F5` in the editor
- save the map with `Ctrl` + `S`, then try `Esc`, `Enter` in the editor

i don't understand what the issue is

**Reply by nobbele** [reply]
ok z,z,f5 then c after is decent but it's a bit jarring to shift around in the timeline so much. There should be something like shift+f5 to start from beginning

**Reply by Walavouchey** [reply]
you say it's jarring but isn't it exactly the same as in stable? 

> in stable I'd just scroll forward a bit (to get to the last note) and click play to make it start from the beginning

i.e. `V`, `V`, `F5`

**Reply by nobbele** [reply]
not when mapping at the tail of the map which is usually the case for me
